app/src/main/java/edu/uc/kovaciad/slicetracker/ui/main/OverviewFragment.kt

```diff
@@ -13,13 +13,12 @@ import androidx.lifecycle.Observer
 import androidx.lifecycle.ViewModel
 import edu.uc.kovaciad.slicetracker.R
 
-class OverviewFragment : Fragment() {
+class OverviewFragment : SuperFragment() {
 
     companion object {
         fun newInstance() = OverviewFragment()
     }
 
-    private lateinit var viewModel: MainViewModel
```

Comment:
removing the instantiation of viewmodel in subFragment now that it is initialized in the super class and can be used within subFragments without creating another variable